packages/core/styles/components/_dialog.scss

```
@@ -44,11 +46,15 @@ dialog {
   [data-actions] {
     display: flex;
     align-items: center;
+    justify-content: flex-end;
```

Stylelint (17.12.0)

[error] Duplicate property "justify-content" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)